registration/views.py

```diff
@@ -44,7 +44,7 @@ def _validation_errors(request, template_name, form) -> http.HttpResponse:
 
 
 def _no_profile_error(request) -> http.HttpResponseRedirect:
-    message = f'There is no profile for {request.user}'
+    message = _(f'There is no profile for {request.user}')
```

Comment:
This looks like a parametrized message ID. If you just use a format-string, it would require a separate translation for each `request.user`.